ArraysLab.js

```diff
@@ -0,0 +1,212 @@
+// # Arrays Lab
```

Comment:
Try to solve more of those problems at the bottom. I think you can solve some of them if you spend time on them. If you have any questions about them, reach out to me or Alejo.